### PR TITLE
Various doc fixes

### PR DESCRIPTION
**Pull request by nodeg**
I updated files regarding the documentation and fixed some spelling and grammer errors. Furthermore I tested the building of HTML, PDF (LaTeX), man pages files and fixed the broken link.

### CODE REVIEW

**Review by SchoolGuy — APPROVED** [review]
LGTM in my eyes

File: docs/virtual-networking-setup.rst [hunk 1]
```diff
@@ -21,24 +21,23 @@ can still create your own. xenbr0 is created in most versions of RHEL by xend st
...
 
-If a network bridge already exists, koan will be able to use that, though in some cases, you'll have to create your own
-bridge in order to get koan to work. You do this by modifying the network configuration on your virtual host, and then
-using the koan parameter ``--virt-bridge=bridgename``.
+If a network bridge already exists, Koan will be able to use that, though in some cases, you'll have to create your own
+bridge in order to get Koan to work. You do this by modifying the network configuration on your virtual host, and then
+using the Koan parameter ``--virt-bridge=bridgename``.
 
 (As we mentioned, if you use virbr0, it's a fake bridge, so be aware you won't be able to ssh into your guests...
 However koan can use that if you REALLY want to)
```
Comment (dansou901) [inline docs/virtual-networking-setup.rst:32]: When you capitalize koan everywhere, you need to do it here as well

**Review by dansou901 — CHANGES_REQUESTED** [review]
See inline comment

File: docs/virtual-networking-setup.rst (hunk 1 above)
Comment (nodeg) [reply docs/virtual-networking-setup.rst]: Thanks! It seems that I missed some of them. Hopefully this is fixed now.

File: docs/koan-on-openvz.rst [hunk 2]
```diff
@@ -13,7 +13,7 @@ Current support for OpenVZ is rather basic, but I think this functionality can r
 How to use it?
 ##############
 
-Because OpenVZ container is in nature chrooted environment we use cobbler+koan to create this on OpenVZ-enabled node.
+Because OpenVZ container is in nature chrooted environment we use Cobbler + Koan to create this on OpenVZ-enabled node.
 For cobbler and Koan in case of OpenVZ all operations is similar - we should define distros, automated installation
```
Comment (dansou901) [inline docs/koan-on-openvz.rst:17]: missing capitilization of Cobbler

**Review by dansou901 — CHANGES_REQUESTED** [review]
Missed one, see inline comment

File: docs/koan-on-openvz.rst (hunk 2 above)
Comment (nodeg) [reply docs/koan-on-openvz.rst]: Fixed.

**Review by dansou901 — APPROVED** [review]
LGTM now.

**Pull request merged**